lib/Test/Selenium/Remote/Driver.pm

```diff
@@ -44,8 +44,12 @@ sub has_args {
     my $self          = shift;
     my $fun_name      = shift;
     my $hash_fun_args = {
-        'find_element'     => 1,
-        'find_elements'     => 1,
+        'find_element'     => 2,
+        'find_no_element' => 2,
+        'find_child_element'     => 3,
+        'find_child_elements'     => 3,
+        'find_element'     => 2,
+        'find_elements'     => 2,
         'compare_elements' => 2,
```

Comment:
Yeah :) 
We could try and migrate T::S::R::D on a named parameter version, since this class seems to be used by few. Something to add on the roadmap at some point ?